ui/admin/app/styles/app.scss

```diff
@@ -735,6 +735,17 @@
   /* stylelint-enable selector-class-pattern */
 }
 
+.config-tags {
+  /* stylelint-disable selector-class-pattern */
+  .hds-table__tbody {
+    .hds-table__td:last-child {
+      // remove width and white-space styles
+      width: unset;
+    }
+  }
+}
+/* stylelint-enable selector-class-pattern */
```

Comment:
Wait is this to unset our _own_ styles? We should nest this under `.list-wrapper-field` if so. This should be something like `&.config-tags`

Reply:
@ZedLi got ya I see your point